build-package/certifaiReferenceModelServer/bank_marketing/decisionTree/Makefile

```diff
@@ -12,7 +12,7 @@ PAYLOAD = '{"$$ref": ${DATASET_NAME}, "model_name": ${MODEL_NAME} }'
 
 test-endpoint:
 	curl --fail -X POST -H "Content-Type: application/json" \
-	  -d @test/test_instances.json ":5111/${MODEL_NAME}/predict"
+	  -d @test/test_instances.json "0.0.0.0:5111/${MODEL_NAME}/predict"
```

Comment:
I just fired up a quick flask server to check and it resolves to `127.0.0.1`.
Just make sure the binding address passed [here](https://github.com/CognitiveScale/certifai-reference-models/blob/bb0af8459bb287bb6f525b612537a8e96b4b7207/build-package/certifaiReferenceModelServer/start.py#L116) is `0.0.0.0`.
If that's the case, then probably something with `/etc/hosts` file?

Reply:
I'm not sure the `/etc/hosts` files are the same. I see that 0.0.0.0 is used, but when running `make test` on the M1 (develop branch) I get this error: 
```
$ make test
cd certifaiReferenceModelServer/all && make test-all-endpoints
testing bank_marketing
testing 'bank-marketing-all' -decisionTree
curl --fail -X POST -H "Content-Type: application/json" \
	  -d @test/test_instances.json ":5111/"bank_marketing_dtree"/predict"
curl: (3) URL using bad/illegal format or missing URL
make[3]: *** [test-endpoint] Error 3
make[2]: *** [test-all] Error 1
make[1]: *** [test-all-endpoints] Error 1
make: *** [test] Error 2
```

(I'm guessing its because a newer version of curl is installed on the laptop.)